internal/api/courses/users/get.go

```diff
@@ -0,0 +1,30 @@
+package users
+
+import (
+	"github.com/edulinq/autograder/internal/api/core"
+)
+
+type GetRequest struct {
+	core.APIRequestCourseUserContext
+	core.MinCourseRoleOther
+
+	TargetCourseUser core.TargetCourseUserSelfOrAdmin `json:"target-email"`
```

Comment:
Move this down to `core.TargetCourseUserSelfOrGrader` (instead of admin),
since graders will sometimes need to do things like this.